server/src/main/java/io/littlehorse/common/model/getable/core/usertaskrun/UserTaskRunModel.java

```diff
@@ -99,6 +100,7 @@ public UserTaskRunModel(
         this.nodeRunId = nodeRunModel.getObjectId();
         this.id = new UserTaskRunIdModel(nodeRunId.getWfRunId());
         this.scheduledTime = new Date();
+        this.epoch = new Date().getTime();
```

Comment:
Ah, let's please just start with `0` and increment it each time something changes.